### DISCUSSION TITLE
APP_URL

### DISCUSSION THREAD

**Discussion by Dunkledore** [firefly-iii/firefly-iii#4719] [General]
In the documentation one is instructed to set APP_URL: https://docs.firefly-iii.org/firefly-iii/installation/docker/#docker-and-reverse-proxies

But in the example env file (right at the bottom) one is told setting this is a waste of time. : https://raw.githubusercontent.com/firefly-iii/firefly-iii/main/.env.example

Am I misinterpreting something?

**Comment by JC5** [comment]
It says right there in the documentation why it's pointless?

**Reply by Dunkledore** [reply]
I missed it, my apologies
